README.md

```diff
@@ -509,7 +493,7 @@ var scope = nock('http://www.example.com')
 You can specify the reply headers like this:
 
 ```js
-var scope = nock('http://www.headdy.com')
+const scope = nock('http://www.headdy.com')
```

Comment:
Same thing for headdy.com, is it something related to Nock?

Reply:
It sounds like a hostname made up for testing :)